File: cmd/debtdrone/testdata/dirty_code/complex.kt
fun complexFunction(x: Int) {
    if (x > 0) {
        for (i in 0 until x) {
            if (i % 2 == 0) {
                if (i % 3 == 0) {
                    for (j in 0 until i) {
                        if (j > 5) {
                            println("Deeply nested")
                        }
                    }
                }
            }
        }
    } else {
        if (x < -5) {
            println("Negative")
        }
    }
}

File: cmd/debtdrone/testdata/dirty_code/complex.go
package main

import "fmt"

func complexFunction(x int) {
	if x > 0 {
		for i := 0; i < x; i++ {
			if i%2 == 0 {
				if i%3 == 0 {
					for j := 0; j < i; j++ {
						if j > 5 {
							fmt.Println("Deeply nested")
						}
					}
				}
			}
		}
	} else {
		if x < -5 {
			fmt.Println("Negative")
		}
	}
}
